### DISCUSSION TITLE
Live Chart is not showing in UI

### DISCUSSION THREAD

**Discussion by Emianna** [unoplatform/uno#17938] [General]
I am using LiveChartsCore.SkiaSharpView.WinUI  for plotting a chart. The chart is not rendering in the UI when we host the web application in a virtual machine and trying to access the same from a remote machine using  its ip address. But the same is working if we browse from the hosted machine using localhost. Can you help me here? 

@agneszitte

**Comment by francoistanguay** [comment]
We need a lot more details than this to help.

We need a repro project that has the error, details about your setup and hopefully we can investigate then.

**Reply by tomchoondal** [reply]
I have added a detailed bug on this link with sample code and screenshots. Can you check this
https://github.com/beto-rodriguez/LiveCharts2/issues/1569

**Reply by francoistanguay** [reply]
Are you able to test over HTTPS vs HTTP? My understanding is that the Mono for WebAssembly runtime has some limitations on HTTP and might expect a secure channel to be provided.

**Reply by MartinZikmund** [reply]
As mentioned in the linked issue:

> There is a bug in SKXamlCanvas when the browser is not in a secure context. We'll work with MS to resolve it. Switch to HTTPS for now.

Closing this discussion for now

**Reply by Emianna** [reply]
> Are you able to test over HTTPS vs HTTP? My understanding is that the Mono for WebAssembly runtime has some limitations on HTTP and might expect a secure channel to be provided.

Thank you for your reply. Will try with HTTPS
